css/properties/letter-spacing.json

```
@@ -42,7 +42,7 @@
             "deprecated": false
           }
         },
-        "svg": {
+        "svg_elements": {
           "__compat": {
             "description": "SVG support",
```

This is more consistent with the rest.

```suggestion
            "description": "On SVG elements",
```